packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -184,6 +210,8 @@ watch(() => props.height, (newHeight) => {
   heightRef.value = newHeight
 })
 
+const rawChartData = computed(() => toRef(props, 'chartData'))
```

Comment:
You don't need to wrap this in a `computed` -- you can just say `composables.useChartSelectedRange(toRef(props, 'chartData'))`.  (`toRef` takes care of ensuring that you still have a reactive thing to pass to the function.)